README.md

```diff
@@ -81,12 +81,18 @@ flutter pub get
 * Connect a physical iOS device or select *Open iOS Simulator* from the device drop-down menu to power on Xcode's simulator.
 * Select your device from the dropdown list and hit the play button (*^R*).
 
+
+|     :bulb:    | Running the app in debug mode is recommended for testing. To do this, run `./run-ios-flavor.bash dev`. Similarly, if you need to run the release version **(not recommended)**, you can run `./run-ios-flavor.bash prod`.|
+|---------------|:---------------------------------------------------------------------------------------------------------------------------------------------|
+
+* You can find more information about build flavours in Flutter [here](https://medium.com/@animeshjain/build-flavors-in-flutter-android-and-ios-with-different-firebase-projects-per-flavor-27c5c5dac10b).
```

Comment:
Move this to CONTRIBUTING

Reply:
@IoanaAlexandru which part, the one about `more information`?

Reply:
Yep, thanks